thoth-api/src/model/series/mod.rs

```diff
@@ -151,7 +151,9 @@ impl fmt::Display for SeriesWithImprint {
         write!(
             f,
             "{} ({}, {})",
```

Comment:
this would display "series (, )" if neither ISSN was set. try to conditionally output the ISSNs, e.g.

```rust
impl fmt::Display for SeriesWithImprint {
    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
        write!(f, "{}", self.series_name)?;

        let issns: Vec<String> = vec![
            self.issn_print.as_ref().map(|s| s.clone()),
            self.issn_digital.as_ref().map(|s| s.clone()),
        ]
        .into_iter()
        .flatten()
        .collect();

        if !issns.is_empty() {
            write!(f, " ({})", issns.join(", "))?;
        }

        Ok(())
    }
}
```

Reply:
And a few tests:

```rust
    #[test]
    fn test_display_with_issns() {
        let series = SeriesWithImprint {
            series_name: String::from("Test Series"),
            issn_print: String::from("1234-5678"),
            issn_digital: String::from("8765-4321"),
            ..Default::default()
        };

        let formatted = format!("{}", series);
        assert_eq!(formatted, "Test Series (1234-5678, 8765-4321)");
    }

    #[test]
    fn test_display_with_single_issn() {
        let series = SeriesWithImprint {
            series_name: String::from("Test Series"),
            issn_print: String::from("1234-5678"),
            ..Default::default()
        };

        let formatted = format!("{}", series);
        assert_eq!(formatted, "Test Series (1234-5678)");
    }

    #[test]
    fn test_display_without_issns() {
        let series = SeriesWithImprint {
            series_name: String::from("Test Series"),
            ..Default::default()
        };

        let formatted = format!("{}", series);
        assert_eq!(formatted, "Test Series");
    }
    ```